azure/iac/master-compliance-test.json

```diff
@@ -2674,6 +2674,130 @@
+                    "tags": [
+                        {
+                            "cloud": "git",
+                            "compliance": [
+                                "Best Practices"
+                            ],
+                            "service": [
+                                "arm"
+                            ]
+                        }
+                    ]
+                },
+                {
+                    "masterTestId": "TEST_postgreSQL_2",
+                    "masterSnapshotId": [
+                        "ARM_TEMPLATE_SNAPSHOT"
+                    ],
+                    "type": "rego",
+                    "rule": "file(postgreSQL.rego)",
+                    "evals": [
+                        {
+                            "id": "PR-AZR-0124-ARM",
+                            "eval": "data.rule.sslEnforcement",
+                            "message": "data.rule.sslEnforcement_err",
+                            "remediationDescription": "Make sure PostgreSQL has ( properties.sslEnforcement = 'Enabled' )",
+                            "remediationFunction": "PR-AZR-0124-ARM.py"
+                        }
+                    ],
+                    "severity": "High",
+                    "title": "Ensure 'Enforce SSL connection' is set to 'ENABLED' for PostgreSQL Database Server",
+                    "description": "Enable SSL connection on PostgreSQL Servers. Rationale: SSL connectivity helps to provide a new layer of security, by connecting database server to client applications using Secure Sockets Layer (SSL). Enforcing SSL connections between database server and client applications helps protect against 'man in the middle' attacks by encrypting the data stream between the server and application.",
+                    "tags": [
+                        {
+                            "cloud": "git",
+                            "compliance": [
+                                "Best Practices"
+                            ],
+                            "service": [
+                                "arm"
+                            ]
+                        }
+                    ]
+                },
+                {
+                    "masterTestId": "TEST_DBACCOUNTS",
+                    "masterSnapshotId": [
+                        "ARM_TEMPLATE_SNAPSHOT"
+                    ],
+                    "type": "rego",
+                    "rule": "file(dbAccounts.rego)",
+                    "evals": [
+                        {
+                            "id": "PR-AZR-0105-ARM",
+                            "eval": "data.rule.tagsLength",
+                            "message": "data.rule.tagsLength_err",
+                            "remediationDescription": "Make sure CosmosDbAccount has ( tags length() > 0 )",
```

Comment:
**See my validation message at line 2496 (for id PR-AZR-0109-ARM).** make the description format like this one.